protocol/answer/negativeAnswer/error.json

```diff
@@ -3,26 +3,24 @@
 	"$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/answer/negativeAnswer/error.json",
 	"title": "If an error occurred, the \"result\" field is absent and the error object must adhere to the following (extensible) format",
 	"description": "In case of negative answer, error generated by the client query",
-
 	"type": "object",
 	"properties": {
-		"code": { 
-			"description": "[Int] error code", 
-			"type": "integer", 
-			"minimum": -5, "maximum": -1,
-			"$comment": "Note: should be in { -5, -4, ..., -1 }",
-			"note": [
-				"0  operation was successful (should never be used)",
-				"-1 invalid action",
-				"-2 invalid resource (e.g. channel does not exist, channel was not subscribed to, etc.)",
-				"-3 resource already exists (e.g. lao already exists, channel already exists, etc.)",
-				"-4 request data is invalid (e.g. message is invalid), use the data field object to provide extra information",
-				"-5 access denied (e.g. subscribing to a “restricted” channel)"
-			]

```

Comment:
where are you moving this? 
Also, there's a `-6` code, equivalent to a `500 internal server error`